lib/dev_wizard/github_gateway/issue.ex

```diff
@@ -1,6 +1,8 @@
 defmodule DevWizard.GithubGateway.Issue do
   alias DevWizard.GithubGateway.{User, Comment, Milestone}
 
+  @linked_issue_regex ~r/(closes|connects)\s+thinkthroughmath\/storyboard#([0-9]+)/i
```

Comment:
You may want to expand this list a bit. I know there are other keywords Waffle supports and I definitely see people on our team use (at the minimum) "fixes."